ubuntudesign/cloud-vanilla-theme
# Add button transition

## QA
- Run `gulp build`
- Check out the demo at demo/index.html
- Go down to the buttons and hover
- You should see a small transition
